defcon/plugins/alertmanager.py

```diff
@@ -109,16 +135,18 @@ def render(template, data):
         env = jinja2.Environment()
         return env.from_string(template).render(**data)
 
-    def _to_status(self, root_alert, block, alert):
+    def _to_status(self, alert):
         """Return a status or None."""
         logging.debug('Handling %s' % (alert))
         if not self.match_labels(alert['labels'], self.labels):
             return None
         status = alert.get('status', {})
-        if self.receiver is not None:
-            if 'routeOpts' in block:
-                if block['routeOpts']['receiver'] != self.receiver:
-                    return None
+
+        # if self.receiver is not None:
+        #     if 'routeOpts' in block:
+        #         if block['routeOpts']['receiver'] != self.receiver:
```

Comment:
Is there a reason to remove this code ?
Also, don't comment code

Reply:
old api was like:
```
...
            "receiver": "default",
            "groupBy": [
              "service",
              "alertname",
              "cluster"
            ],
            "groupWait": 30000000000,
            "groupInterval": 300000000000,
            "repeatInterval": 10800000000000
          },
          "alerts": [
...
```
i'll remove